plasmapy/analysis/swept_langmuir/floating_potential.py

```diff
@@ -38,8 +38,7 @@ def find_floating_potential(
     ----------
 
     voltage: `numpy.ndarray`
-        1-D numpy array of monotonically ascending/descending probe biases
-        (should be in volts)
+        1-D numpy array of monotonically ascending probe biases (should be in volts)
 
     current: `numpy.ndarray`
         1-D numpy array of probe current (should be in amperes) corresponding
```

Comment:
```suggestion
        1-D numpy array of monotonically ascending probe biases (must be in volts)

    current: `numpy.ndarray`
        1-D numpy array of probe current (must be in amperes) corresponding
```
[optional] If the function requires volts and amperes here, then we could be more specific with the language.  If not, feel free to ignore.

Reply:
I have to think about this. `voltage` and `current` should be in volts and amperes, respectively, but it is not needed for this function.  The Isat can be determined regardless of the units; it's just the returned Isat will share the same equivalent units as `voltage` and `current`.

Reply:
I wrote this thinking about $I_{sat}$, but the statement is still true for $V_f$.